src/main/java/lotto/domain/LottoResult.java

```diff
@@ -0,0 +1,42 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+
+public class LottoResult {
+    private static final int DEFAULT_VALUE = 0;
+    private static final int PLUS_VALUE = 1;
+
+    private final Map<LottoReward, Integer> resultMap;
+
+    public LottoResult(final List<LottoReward> lottoRewards) {
+        resultMap = new EnumMap<>(LottoReward.class);
+        Arrays.stream(LottoReward.values()).forEach(lottoReward -> resultMap.put(lottoReward, DEFAULT_VALUE));
```

Comment:
하단과 같이 표현해볼수도 있겠네요!

```suggestion
        for (LottoReward reward : LottoReward.values()) {
           resultMap.put(reward, Collections.frequency(lottoRewards, reward));
        }
```

추가로 calculateProfit 등의 기능이 올바르게 작동하는지 테스트를 추가해보는것도
좋을것 같습니다.